Source/Particles/MultiParticleContainer.cpp

```diff
@@ -242,27 +241,17 @@ MultiParticleContainer::ReadParameters ()
                 }
             }
 
-            // collision
+            // binary collisions
             ParmParse pc("collisions");
-            pc.query("ncollisions", ncollisions);
-            AMREX_ALWAYS_ASSERT(ncollisions >= 0);
-            if (ncollisions > 0) {
-                pc.getarr("collision_names", collision_names);
-                AMREX_ALWAYS_ASSERT(collision_names.size() == ncollisions);
-            }
+            pc.queryarr("collision_names", collision_names);
```

Comment:
as a follow-up, we could potentially rename this to `collisions.names` for consistency.